__tests__/distributors/base.test.ts

```diff
@@ -0,0 +1,180 @@
+import * as tc from '@actions/tool-cache';
+import * as core from '@actions/core';
+
+import path from 'path';
+import * as semver from 'semver';
+
+import { JavaBase } from '../../src/distributors/base-installer';
+import {
+  JavaDownloadRelease,
+  JavaInstallerOptions,
+  JavaInstallerResults
+} from '../../src/distributors/base-models';
+
+class EmptyJavaBase extends JavaBase {
+  constructor(installerOptions: JavaInstallerOptions) {
+    super('Empty', installerOptions);
+  }
+
+  protected downloadTool(javaRelease: JavaDownloadRelease): Promise<JavaInstallerResults> {
+    throw new Error('Method not implemented.');
+  }
+
+  protected findPackageForDownload(range: semver.Range): Promise<JavaDownloadRelease> {
+    throw new Error('Method not implemented.');
+  }
+}
+
+describe('Test findInToolcache', () => {
```

Comment:
Use name `findInToolcache` for describe. The same for other describes

Reply:
Done. Thank you.